recipes/slda/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export USE_CYTHON=True
+
+export CC=gcc CXX=g++
```

Comment:
So, this is going to cause issues if we do this for both platforms. I expect either this isn't needed or we need to use something like the `toolchain`.